packages/razzle/scripts/build.js

```diff
@@ -74,8 +74,12 @@ function build(previousFileSizes) {
   let razzle = {};
   try {
     razzle = require(paths.appRazzleConfig);
+    console.log(chalk.yellow('Found custom razzle.config.js'));
```

Comment:
Yellow should be reserved for warnings.
```suggestion
    console.log(chalk.dim('Found custom razzle.config.js'));
```